### PR TITLE
feat/local flash notice added in seprate locale yml

### PR DESCRIPTION
**Pull request by vasanthakumar-a**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -12,22 +12,16 @@
...
+      toastr.options = {
+        "closeButton": true
+      };
+      <% if flash[:notice] %>
+        toastr["success"]("<%= flash[:notice] %>")
+      <% end %>
+      <% if flash[:alert] %>
+        toastr["warning"]("<%= flash[:alert] %>")
+      <% end %>
       </script>
```
Comment (iMagesh) [inline app/views/layouts/application.html.erb:25]: can we move this to a separate partial?

File: config/locales/flash.en.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  left_community: "You have left the Community 💔"
+  post_save: "Post saved!"
+  post_unsave: "Post Unsaved!"
+  post_edit: "Post Edited successfully"
+  post_closed: "Post Discussion Closed Successfully"
+  post_destroyed: "Post Destroyed Successfully"
+  user_not_prest: "Username '%{username}' not present in Community"
+  admin_ban: "The user '%{username}' is Admin"
+  already_banned: "The user '%{username}' is already banned"
+  not_present_to_ban: "The user '%{username}' is not present in "
```
Comment (iMagesh) [inline config/locales/flash.en.yml:20]: can we group this as error messages and success messages? under notice

**Comment by iMagesh** [comment]
@vasanthakumar-a let me know once you have modified this

File: config/locales/flash.en.yml (hunk 2 above)
Comment (vasanthakumar-a) [reply config/locales/flash.en.yml]: grouped the flash messages based on category

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (vasanthakumar-a) [reply app/views/layouts/application.html.erb]: this moved to a separate partial file @iMagesh

**Comment by vasanthakumar-a** [comment]
Resolved the conflicts to be up to date with master

**Pull request merged**